content/en/docs/tasks/manage-daemon/create-daemon-set.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Building a Basic DaemonSet  
+content_type: task  
+weight: 30  
```

Comment:
```suggestion
weight: 5
```
As this page explains about building a basic DaemonSet, it must be the first task.